{
  "pack_xdsl_access_diagnostic_title": "Diagnostic de votre ligne",
  "pack_xdsl_access_diagnostic_wizard_title": "Assistente risoluzione dei problemi",
  "pack_xdsl_access_diagnostic_launch": "Avvia una diagnostica",
  "pack_xdsl_access_diagnostic_assistant_launch": "Avvia l'assistente",
  "pack_xdsl_access_diagnostic_remaining": "{{value}} rimanenti",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme1": "La diagnostica della linea indica un problema di sincronizzazione.",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme2": "Ho una perdita totale di connessione.",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme3": "Il mio operatore rileva una connessione ma non riesco a connettermi.",
  "pack_xdsl_access_diagnostic_general": "Report sullo stato della linea",
  "pack_xdsl_access_diagnostic_time": "Orario della diagnostica",
  "pack_xdsl_access_diagnostic_access_incident_ongoing": "Risoluzione di un incidente OVHcloud in corso...",
  "pack_xdsl_access_diagnostic_access_works": "Nessun incidente",
  "pack_xdsl_access_diagnostic_access_fails": "Incidente",
  "pack_xdsl_access_diagnostic_modem_connected": "Stato della connessione",
  "pack_xdsl_access_diagnostic_modem_state_disconnected": "Il tuo dispositivo OVHcloud non è connesso",
  "pack_xdsl_access_diagnostic_modem_state_connected": "Il tuo dispositivo OVHcloud è connesso",
  "pack_xdsl_access_diagnostic_modem_state_running": "Test del dispositivo in corso...",
  "pack_xdsl_access_diagnostic_active_on_lns": "Gateway in uscita LNS",
  "pack_xdsl_access_diagnostic_lns_active_ok": "Il tuo dispositivo è attivo",
  "pack_xdsl_access_diagnostic_lns_active_ko": "Il tuo dispositivo non è attivo",
  "pack_xdsl_access_diagnostic_ping": "Risposta ai ping",
  "pack_xdsl_access_diagnostic_ping_ko": "Il tuo dispositivo non risponde ai ping",
  "pack_xdsl_access_diagnostic_ping_ok": "Il tuo dispositivo risponde correttamente ai ping",
  "pack_xdsl_access_diagnostic_ping_running": "Ping in corso...",
  "pack_xdsl_access_diagnostic_line": "Linea n°{{name}}",
  "pack_xdsl_access_diagnostic_line_sync": "Sincronizzazione della linea",
  "pack_xdsl_access_diagnostic_line_sync_ko": "La linea non è sincronizzata",
  "pack_xdsl_access_diagnostic_line_sync_ok": "La linea è sincronizzata",
  "pack_xdsl_access_diagnostic_line_proposed_profile_id": "Profilo di connessione proposto",
  "pack_xdsl_access_diagnostic_line_current_profile_id": "Profilo di connessione corrente",
  "pack_xdsl_access_diagnostic_line_line_test": "Test della linea",
  "pack_xdsl_access_diagnostic_line_line_test_actionPending": "In corso",
  "pack_xdsl_access_diagnostic_line_line_test_customerSideProblem": "Problema lato client",
  "pack_xdsl_access_diagnostic_line_line_test_error": "Errore",
  "pack_xdsl_access_diagnostic_line_line_test_noProblem": "Funzionamento OK",
  "pack_xdsl_access_diagnostic_line_line_test_ovhSideProblem": "Problema lato OVHcloud",
  "pack_xdsl_access_diagnostic_line_error_diag_launch": "Impossibile avviare la diagnostica.",
  "pack_xdsl_access_diagnostic_line_error_diag_poll": "Si è verificato un errore durante la diagnostica."
}
